code/frameworks/nextjs/package.json

```diff
@@ -141,12 +141,15 @@
       "./src/preview.tsx",
       "./src/next-image-loader-stub.ts",
       "./src/images/context.ts",
+      "./src/images/decorator.tsx",
       "./src/images/next-future-image.tsx",
       "./src/images/next-legacy-image.tsx",
       "./src/images/next-image.tsx",
       "./src/font/webpack/loader/storybook-nextjs-font-loader.ts"
     ],
     "externals": [
+      "./context",
+      "./images/decorator",
```

Comment:
I'm skeptical if this will work.

Your PR description seems to assume this is being externalized by webpack.. But this config isn't for webpack, it's for `tsup`.

Reply:
Thank you for checking in!

I was indeed under the impression that webpack was being used, but as far as I can tell it seems like `tsup` does support specifying externals, which should resolve this issue. Running `yarn task` and building locally results in what should be the correct output I've posted above.

Also, if you open dev tools and check the [next branch's `Eager` story](https://634ff0454a50407deedde524-vxzgsqcetx.chromatic.com/?path=/story/frameworks-nextjs-image--eager) you can see the image is being lazily loaded, while [this branch's `Eager` story](https://634ff0d0ec053b270775979d-ckcpybetvq.chromatic.com/?path=/story/frameworks-nextjs-image--eager) is correctly being eagerly loaded (the `Eager` story's snapshots should change due to this, but it seems the image element has been chromatic-ignored, thus not triggering a change).